tests/phpunit/class-wc-stripe-checkout-sessions-ajax-handler-test.php

```
@@ -25,6 +31,135 @@ public function test_init_hooks(): void {
+				];
+			}
+
+			return $return_value;
+		};
+
+		add_filter( 'pre_http_request', $test_request, 10, 3 );
+
+		$_REQUEST['_ajax_nonce'] = $is_valid_nonce ? wp_create_nonce( 'wc_stripe_get_checkout_session_status_nonce' ) : 'invalid_nonce_value';
+		$_POST['session_id']     = $session_id;
+
+		$ajax_handler = new WC_Stripe_Checkout_Sessions_Ajax_Handler();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '69', column '23'). (undefined)

(MissingImport)